habitat/sims/habitat_simulator/habitat_simulator.py

```diff
@@ -310,6 +315,14 @@ def create_sim_config(
                     "type",
                     "width",
                 },
+                trans_dict={
+                    "sensor_model_type": lambda v: getattr(
+                        habitat_sim.FisheyeSensorModelType, v
```

Comment:
Maybe let's use `fisheye_model_type` or `fisheye_type` instead of general `sensor_model_type` field that maps in specific FisheyeSensorModelType.

Reply:
That would a refactor to Habitat-Sim, @mathfac sadly. So best for a seperate double PR.

Reply:
No, it's a pass through to Habitat-Sim's Fisheye SensorSpec.

Reply:
Got it, then ignore my comment.